Library/Homebrew/brew.sh

```diff
@@ -267,6 +271,12 @@ access to all bottles."
 EOS
 fi
 
+if [[ -z "$HOMEBREW_ENV_PRUNED" ]]
```

Comment:
I like the double-guard here but probably warrants a comment explaining this is what it's doing.